src/clj/pyregence/cameras.clj

```diff
@@ -207,16 +207,16 @@
   "Builds a GeoJSON response of the current wildfire cameras."
   []
   (data-response
-    (if (valid-cache?)
-      @camera-cache
-      (let [new-cameras (->> (concat (get-and-conform-wildfire-cameras!)
-                                     (get-and-conform-california-cameras!))
+   (if (valid-cache?)
+     @camera-cache
+     (let [new-cameras (->> (concat (get-and-conform-wildfire-cameras!)
+                                    (get-and-conform-california-cameras!))
```

Comment:
What's the reason for only having one space (instead of 2) before the sole argument to `data-response` here?

Reply:
I see! You learn something new every day.